src/user-utils.ts

```diff
@@ -0,0 +1,12 @@
+import { IMParticleUser } from './identity-user-interfaces';
+
+export function didUserChange(
```

Comment:
I think the context here is how the user changes. For `didUserChange` we're comparing previous and new user mpid and `isLoggedIn` values, but in `hasUserChanged` we're technically comparing whether the `mpid` of the user and Identity API Data is different.

This may be unnecessarily verbose, but perhaps we can rename them to `hasMPID` and `hasMPIDAndUserLoginChanged`?